biblio-core.el

```diff
@@ -859,25 +894,27 @@ Get prompt string from BACKEND."
   (let* ((prompt (funcall backend 'prompt)))
     (read-string prompt nil 'biblio--search-history)))
 
-(defun biblio--lookup-1 (backend query)
-  "Just like `biblio-lookup' on BACKEND and QUERY, but never prompt."
-  (let ((results-buffer (biblio--make-results-buffer (current-buffer) query backend)))
+(defun biblio--lookup-1 (backend query target)
+  "Just like `biblio-lookup' on BACKEND, QUERY and TARGET, but never prompt."
+  (let ((results-buffer (biblio--make-results-buffer target query backend)))
     (biblio-url-retrieve
      (funcall backend 'url query)
      (biblio--callback results-buffer backend))
     results-buffer))
 
 ;;;###autoload
-(defun biblio-lookup (&optional backend query)
-  "Perform a search using BACKEND, and QUERY.
+(defun biblio-lookup (&optional backend query target)
+  "Perform a search using BACKEND, QUERY and TARGET buffer.
 Prompt for any missing or nil arguments.  BACKEND should be a
 function obeying the interface described in the docstring of
-`biblio-backends'.  Returns the buffer in which results will be
-inserted."
+`biblio-backends'. TARGET is the buffer into which bibliographic
+entries will be inserted. Returns the buffer in which results
```

Comment:
It might be nice to add something to the doc string about what happens if `target` is nil, i.e., refer to `biblio-target-buffer-functions` to control the behavior of how `target` is chosen when passed as `nil`.